examples/example.css

```diff
@@ -1,3 +1,21 @@
 textarea {
   tab-size: 1; /* Set tab size to 2 spaces */
+}
+
+.boxplot .target .median {
+	stroke: #ff9315;
+}
+.boxplot .target line {
+	stroke: #ff9315;
```

Comment:
Why not to allow to pass colors via JS as it is done for other charts?

Reply:
So this isn't a series colroing case, this is just a class of objects i'm introducing for 'target' and 'comparator'.  I wanted this controllable by css/less styling. not set programatically.

Reply:
btw: I didn't put the .compare class in this css because the default bleu worked, but I can push up CSS classes in example.css so that it is clear.